spot-contracts/contracts/PerpetualTranche.sol

```diff
@@ -177,8 +149,8 @@ contract PerpetualTranche is
     uint8 public constant PRICE_DECIMALS = 8;
     uint256 public constant UNIT_PRICE = (10**PRICE_DECIMALS);
 
-    uint8 public constant PERC_DECIMALS = 8;
-    uint256 public constant HUNDRED_PERC = 10**PERC_DECIMALS;
+    uint8 public constant FEE_POLICY_DECIMALS = 8;
+    uint256 public constant FEE_ONE_PERC = (10**FEE_POLICY_DECIMALS);
```

Comment:
naming is a bit awkward. maybe `FEE_UNIT`

Reply:
I like how the existing version explicitly says "One percent", given the previous discussions.

Because "unit" could mean "1.0 as in 1%", but it could also mean "1.0 as in 100%"

Reply:
Yes prefer `FEE_ONE_PERC`. Open better options though ..